src/vmware/azext_vmware/custom.py

```diff
@@ -177,13 +170,19 @@ def hcxenterprisesite_delete(cmd, client: AVSClient, resource_group_name, privat
 
 
 def datastore_create(cmd, client: AVSClient, resource_group_name, private_cloud, cluster, name, nfs_provider_ip=None, nfs_file_path=None, endpoints=[], lun_name=None):
-    from azext_vmware.vendored_sdks.avs_client.models import Datastore, NetAppVolume, DiskPoolVolume
-    datastore = Datastore()
-    if nfs_provider_ip is not None or nfs_file_path is not None:
-        datastore.net_app_volume = NetAppVolume(nfs_provider_ip=nfs_provider_ip, nfs_file_path=nfs_file_path)
-    if len(endpoints) > 0 or lun_name is not None:
-        datastore.disk_pool_volume = DiskPoolVolume(endpoints=endpoints, lun_name=lun_name)
-    return client.datastores.begin_create(resource_group_name=resource_group_name, private_cloud_name=private_cloud, cluster_name=cluster, datastore_name=name, datastore=datastore)
+    print('Please use "az vmware datastore netapp-volume create" or "az vmware datastore disk-pool-volume create" instead.')

```

Comment:
Please add the deprecate declaration on `commands.py` to prompt the discard information, and this logic for creating datastores here can be remained, so as to avoid causing breaking change to the users, such as:
```
 with self.command_group('vmware datastore', vmware_sdk, client_factory=cf_vmware) as g:
        g.custom_command('create', 'datastore_create', deprecate_info=g.deprecate(redirect='"az vmware datastore netapp-volume create" or "az vmware datastore disk-pool-volume create"', hide=True))
```

Reply:
Thanks. I added it.